study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/errorPage.jsp

```diff
@@ -0,0 +1,158 @@
+          content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+    <!-- Web Fonts  -->
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
+
+    <!-- Vendor CSS -->
+    <link rel="stylesheet" href="/studybuilder/vendor/boostrap/bootstrap.min.css">
+  	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css">
+ 
+ 
+    <!-- Theme Responsive CSS -->
+    <link rel="stylesheet" href="/studybuilder/css/layout.css">
+
+    <!-- Theme CSS -->
+    <link rel="stylesheet" href="/studybuilder/css/theme.css">
+    <link rel="stylesheet" href="/studybuilder/css/style.css">
+  
+    <!-- Vendor -->
+    <script src="/studybuilder/vendor/jquery/jquery-3.1.1.min.js"></script>
+    <script src="/studybuilder/vendor/summernote/popper.min.js"></script>
+    <script src="/studybuilder/vendor/boostrap/bootstrap.min.js"></script>
+  
+   
+<style>
+html, body {
+  height: 100%;
+}
+
+ header.header {
+	 width: 100%;
+	 height: 50px;
+	 background-color: #fff;
+	 user-select: none;
+}
+ header.header .navigation {
+	 width: 100%;
+	 height: 100%;
+}
+ header.header .navigation .header {
+	 width: 200px;
+	 height: 100%;
+	 float: left;
+	 display: inline-block;
+}
+
+.page-wrap {
+  min-height: 100vh;
+}
+
+@media only screen and (max-width: 2560px) and (min-width: 1020px) {
+  .custom__font_error {
+    font-size: 140px !important;
+  }
+}
+
+@media only screen and (max-width: 1019px) and (min-width: 760px) {
+  .custom__font_error {
+    font-size: 140px !important;
+  }
+}
+
+@media only screen and (max-width: 760px) and (min-width: 320px) {
+  .custom__font_error {
+    font-size: 140px !important;
+  }
+}
+
+.custom__font_error {
+  color: #b3bcc6 !important;
+  margin-top: -20px;
+}
+
+.custom_sub_font {
+  color: #b3bcc6 !important;
+  margin-top: -20px;
+  font-size: 40px;
+}
+
+ .container, .row {
+  height: 100%;
+  min-height: 100%;
+}
+
+.invalidLink_text{
+	color: #353a3e;
+    font-size: 100%;
+    font-weight: 600; 
+}
+
+.width__50 {
+  width: 50%;
+}
+.mar_0_auto {
+  margin: 0 auto;
+}
+</style>
+  </head>
+  <body>
+   
+    <!-- content start  -->
+<header class="header">
+    <div class="navigation">
+        <div class="header pt-2">
+           <a href="/studybuilder/login.do" class="navbar-brand">
+          <img
+            class="mr-sm width__auto"
+           src="/studybuilder/images/logo/logo_innerScreens.png"
+            alt="main logo"
+          />
+        </a>
+        </div>
+       
+    </div>
+</header>
+
+
+<div class="d-flex align-items-center justify-content-center" style="height: 100vh">
+      <div class="col-md-12 text-center">
+        <span class="display-1 d-block">
+          <img src="/studybuilder/images/icons/ErrorIcon.svg" alt="Page not found here" />
+        </span>
+        <div class="invalidLink_text mt-4 mb-4 width__50  mar_0_auto">This link is no longer valid to be used. 
```

Comment:
This is not proper grammar. Please ask tech writer for help.

Reply:
Fixed